docs/source/glossary.rst

```diff
@@ -1,6 +1,11 @@
 Glossary
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+These definitions have also been put into a form which can be easily learnt/revised using `Anki <https://apps.ankiweb.net/>`_ a cross platform app specifically designed for long term knowledge retention.
+
+- `Introductory video on how to use the flashcards <https://www.youtube.com/watch?v=Dji_h7PILrw>`_
+- `Hugging Face Transformers Glossary Flashcards <https://www.darigovresearch.com/huggingface-transformers-glossary-flashcards>`_
```

Comment:
While I love the idea of having flashcards for our glossary, I think this should be a bit further inside this page, after the definitions of the general terms.
Also, could you make it clearer it's not done by Hugging Face and take full credit for it? Something like "Darigov Research has worked on flashcards for ..."?